_docs/templates/_npe2_writers_guide.md.jinja

```diff
@@ -31,7 +31,7 @@ by the corresponding writer contribution in the manifest).
    **attributes**, (e.g. `{'name': 'My Layer', 'opacity': 0.6}`)
 
    ```python
-   def single_layer_writer(path: str, data: Any, attributes: dict) -> Optional[str]:
+   def single_layer_writer(path: str, data: Any, attributes: dict) -> List[str]:
```

Comment:
I have a hard time remembering where we ended up on this. It looks like types.py also shows `SingleWriterFunction` returning a `List`. So I must be wrong. Looks like the change on 34 is fine.